test/expect/full-real-005.log

```diff
@@ -52,7 +52,6 @@ log-level-console=debug
 log-level-file=trace
 log-level-stderr=off
 log-path=[TEST_PATH]/db-standby/spool/log
-repo-path=[TEST_PATH]/db-master/repo
```

Comment:
In the user-guide we show the repo-path being set in the db-standby pgbackrest.conf file - if it is no longer required, then shouldn't we update the user-guide?